charts/ratify/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+{{- if not (or .Values.notation.enabled .Values.cosign.enabled .Values.sbom.enabled) }}
+NOTES:
```

Comment:
nit: `NOTES` can be removed as this would be auto-generated by helm. Otherwise it will look like:
<img width="485" alt="image" src="https://github.com/user-attachments/assets/bc8b01f6-13b0-413d-8c9b-a1d9918c283c">